projects/electron-ngrx/src/lib/selector-manager.ts

```diff
@@ -0,0 +1,22 @@
+import {h64} from 'xxhashjs';
+
```

Comment:
why isn't it a service? any service you can just convert to methods. Is it right?

Reply:
selector manager cannot be service, because of some functions like "getSelectorByHash" that should be used inside the library but not outside the library but there are other functions like "registerSelector" that should be export by the library, and it's impossible to export from the library half of the service functions.